### PR TITLE
fix repo address and add repo demo

### PR DESCRIPTION
**Pull request by huiwq1990**
/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by huiwq1990** [comment]
@rambohe-ch

File: README.md [hunk 1]
```diff
@@ -17,12 +18,17 @@ Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
...
+
+Install `yurt-app-manager` use command as follows:
+
+```shell
+helm upgrade --install yurt-app-manager openyurt/yurt-app-manager
+```
 
 ## Contributing
 
 The source code of all [Opneyurt](https://openyurt.io) community [Helm](https://helm.sh) charts can be found on Github: <https://github.com/openyurtio/openyurt-helm>
```
Comment (rambohe-ch) [inline README.md]: typo: Opneyurt --> OpenYurt

Comment (huiwq1990) [reply README.md]: done

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**